tests/Broker/BrokerTests.cc

```diff
@@ -66,14 +66,14 @@ TEST(Broker, InvalidCrc)
 
     Broker_register_error_callback(&testErrorDetectedCallback);
 
-    uint8_t data[] = { 0, 0, 0, 0, 0, 1, 0, 0, 0, 0 };
+    uint8_t data[] = { 0, 0, 0, 0, 255, 255, 255, 255, 0, 0, 0, 0 };
```

Comment:
Why this change?

Reply:
I guess the test previously expected 'strict CCSDS' behaviour in some places.  The length field for non strict CCSDS takes 4 bytes and 'zero' value is represented by 0xFFFFFFFF because it is always decremented by 1.

`size_t
readPacketDataLength(const uint8_t* const packetPointer)
{
    const uint32_t packetSize =
            ((uint32_t)(packetPointer[4] << 24u) | packetPointer[5] << 16u | packetPointer[6] << 8u | packetPointer[7]);
    const uint32_t zeroPacketSize = 0xffffffff;
    if(packetSize == zeroPacketSize) {
        return 0;
    }
    return (size_t)packetSize + 1;
}
`